### DISCUSSION TITLE
Does anyone know how to implement this functionality?

### DISCUSSION THREAD

**Discussion by shenxiangwei** [mozilla/hubs#5543] [Art Questions]
There is a function that I want to implement in the hubs. The effect is that when I leave the room of the hubs, I want leave a gradually disappearing image. For example, within 24 hours, it will gradually disappear. I would like to ask if there is any easy way to implement it ?

**Comment by tksham** [comment]
what do you mean by disappearing image? 

having an image with opacity / transparent from 1 to 0 gradually within the 24 hours you mentioned (solid to transparnet)?

or should the image be blocking the room view? 
(the image is gradually displaying from transparent to non-transparent (0 to 1) then blocking people to viewing anything from the room)
